src/main/java/ToDo.java

```diff
@@ -0,0 +1,17 @@
+public class ToDo extends Task{
+    String tag = "[T]";
+    public ToDo() {
+        super.tag = tag;
```

Comment:
Should you make this into a `super(tag)` and add a constructor to the Task class?